types/v56/types.go

```diff
@@ -1360,11 +1360,106 @@ type VM struct {
 	// TODO: OVF Sections to be implemented
 	// Environment OVF_Environment `xml:"Environment,omitempty"
 
+	VmSpecSection *VmSpecSection `xml:"VmSpecSection,omitempty"`
+
 	VMCapabilities *VMCapabilities `xml:"VmCapabilities,omitempty"` // Allows you to specify certain capabilities of this virtual machine.
 	StorageProfile *Reference      `xml:"StorageProfile,omitempty"` // A reference to a storage profile to be used for this object. The specified storage profile must exist in the organization vDC that contains the object. If not specified, the default storage profile for the vDC is used.
 	ProductSection *ProductSection `xml:"ProductSection,omitempty"`
 }
 
+// VM represents a virtual machine only with Disk setting update part
+type VMDiskChange struct {
+	// Attributes
+	XMLName xml.Name `xml:"Vm"`
+	Ovf     string   `xml:"xmlns:ovf,attr,omitempty"`
+	Xsi     string   `xml:"xmlns:xsi,attr,omitempty"`
+	Xmlns   string   `xml:"xmlns,attr,omitempty"`
+
+	HREF string `xml:"href,attr,omitempty"`
+	Type string `xml:"type,attr,omitempty"`
+	Name string `xml:"name,attr"`
```

Comment:
Thinking more about it, I'd like to see all fields of this structure commented (like of structure `VmSpecSection` below). It's a good practice and avoids uncertainties like what does this `Name` mean. Apparently, it's VM's name not disk's, but that is not clear without a comment.

Reply:
added